tests/test_external_account.py

```diff
@@ -67,23 +67,31 @@
 
 VALID_TOKEN_URLS = [
     "https://sts.googleapis.com",
+    "https://sts.mtls.googleapis.com",
     "https://us-east-1.sts.googleapis.com",
+    "https://us-east-1.sts.mtls.googleapis.com",
     "https://US-EAST-1.sts.googleapis.com",
     "https://sts.us-east-1.googleapis.com",
     "https://sts.US-WEST-1.googleapis.com",
```

Comment:
add a case for "https://sts.mtls.US-WEST-1.googleapis.com" (if that is allowed)?

Reply:
IIUC MTLS must prefix googleapis.com but I will check